### PR TITLE
👷‍♀️ FIX: Realms 

### PR DESCRIPTION
**Pull request by kenjinp**
update Realms class to new makeUtils format

### CODE REVIEW

File: src/Nexus.ts [hunk 1]
```diff
@@ -16,6 +18,7 @@ type NexusConfig = {
 // Without this, jest crashes really bad...
 if (process.env.NODE === 'TEST') {
   Organization;
+  Realm;
```
Comment (mdupont) [inline src/Nexus.ts]: Too bad we have to do this for Jest 😢

File: src/Realm/index.ts [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
-  getRealm,
-  listRealms,
-  createRealm,
-  updateRealm,
-  deprecateRealm,
-} from './utils';
-import { CreateOrgPayload } from '../Organization/types';
+import { RealmResponse, CreateRealmPayload, RealmUtils } from './types';
+import store from '../store';
+import Store from '../utils/Store';
```
Comment (mdupont) [inline src/Realm/index.ts]: We ought to find a better way to distinguish them based on more than the capitalization of the first letter...

Comment (kenjinp) [reply src/Realm/index.ts]: yup

File: src/Nexus.ts (hunk 1 above)
Comment (kenjinp) [reply src/Nexus.ts]: yeah I dont understand what's going on. I think in the future we should just dump jest for a more barebones testing framework?

**Pull request merged**